circle.yml

```diff
@@ -7,7 +7,7 @@ machine:
     ADMINS: Olivier Le Floch <olefloch@work4labs.com>
     SECRET_KEY: EamJrjviiW13ue8PEamJrjviiW13ue8P
   python:
-    version: 2.7.9
+    version: 2.7.11
```

Comment:
I would normally agree to this, if they didn't release those significant SSL changes in .9… :-/